selfdrive/car/car_helpers.py

```diff
@@ -141,9 +141,10 @@ def fingerprint(logcan, sendcan, num_pandas):
       cached = True
     else:
       cloudlog.warning("Getting VIN & FW versions")
-      # enable OBD multiplexing for Vin query, also allows time for sendcan subscriber to connect
+      # enable OBD multiplexing for VIN query
+      # NOTE: this takes ~0.1s and is relied on to allow sendcan subscriber to connect
```

Comment:
added comment since we've been relying on this to be reliable for a few months, unknowingly. https://github.com/commaai/openpilot/pull/31172 would guarantee this though